x/authz/authorization_grant.go

```diff
@@ -10,40 +10,55 @@ import (
 )
 
 // NewGrant returns new AuthrizationGrant
```

Comment:
```suggestion
// NewGrant returns new authorization Grant
```
or even get rid of "authorization"?